src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfProperties = sourceString.split(';');
+  let tempArr = [];
+  const resultObj = {};
+
+  arrayOfProperties.forEach(item => {
+    tempArr = item.split(':');
+
+    if (tempArr.length === 2) {
+      resultObj[tempArr[0].trim()] = tempArr[1].trim();
+    }
+  });
```

Comment:
`tempArr ` array looks like not vital here, you can decompose your values right away like:

```suggestion
  const resultObj = {};

  arrayOfProperties.forEach(item => {
    const [key, value] = item.split(':');

    if (key.length && value.length) {
      resultObj[key] = value.trim();
    }
  });
```